docs/bots-guide.md

```diff
@@ -91,14 +91,14 @@ You need:
 
     * In the *Your bots* panel, click on the green icon to download
       its configuration file *.zuliprc* (the structure of this file is
-      explained [here](#configuration-file).
+      explained [here](#configuration-file).)
```

Comment:
(Unless there is a convention for this, parentheses within a sent can't eat a dot.)